ROADMAP.md

```diff
@@ -27,17 +27,17 @@ Note that Barcelona, which the mac-nosip connector uses, is no longer maintained
 | Media/files                      | ✔️  | ✔️        | ✔️          |
 | Replies                          | ✔️  | ✔️        | ✔️          |
 | Tapbacks                         | ✔️  | ✔️        | ✔️          |
-| Edits                            | ❌   | ❌         | ❌           |
-| Unsends                          | ❌   | ❌         | ❌           |
+| Edits                            | ❌   | ❌         | ✔️ (BlueBubbles Server >= 1.9.6)           |
```

Comment:
should we call the server info API endpoint and have the connector startup fail if the version is "too old"?

Reply:
Older versions won't cause bugs or anything, Bluebubbles just won't send the updated message event over the socket unless it is this version. I'd rather not force people to upgrade for something trivial like this. Pulse, this version of bluebubbles is still in alpha for now, I had just been working with their dev to add support.